src/tests/integration/D3MAaveV3.t.sol

```diff
@@ -0,0 +1,1273 @@
+    uint16 id;
+    //aToken address
+    address aTokenAddress;
+    //stableDebtToken address
+    address stableDebtTokenAddress;
+    //variableDebtToken address
+    address variableDebtTokenAddress;
+    //address of the interest rate strategy
+    address interestRateStrategyAddress;
+    //the current treasury balance, scaled
+    uint128 accruedToTreasury;
+    //the outstanding unbacked aTokens minted through the bridging feature
+    uint128 unbacked;
+    //the outstanding debt borrowed against this asset in isolation mode
+    uint128 isolationModeTotalDebt;
+}
+
+interface LendingPoolLike {
+    function deposit(address asset, uint256 amount, address onBehalfOf, uint16 referralCode) external;
+    function borrow(address asset, uint256 amount, uint256 interestRateMode, uint16 referralCode, address onBehalfOf) external;
+    function repay(address asset, uint256 amount, uint256 rateMode, address onBehalfOf) external;
+    function getReserveData(address asset) external view returns (ReserveDataV3 memory);
+}
+
+interface InterestRateStrategyLike {
+    function getMaxVariableBorrowRate() external view returns (uint256);
+    function calculateInterestRates(
+        address reserve,
+        uint256 availableLiquidity,
+        uint256 totalStableDebt,
+        uint256 totalVariableDebt,
+        uint256 averageStableBorrowRate,
+        uint256 reserveFactor
+    ) external returns (
+        uint256,
+        uint256,
+        uint256
+    );
+}
+
+interface ATokenLike is TokenLike {
+    function scaledBalanceOf(address) external view returns (uint256);
+}
+
+interface RewardsClaimerLike {
+    function getRewardsBalance(address[] calldata assets, address user) external view returns (uint256);
+}
+
+contract D3MAaveV3Test is DssTest {
+    VatLike vat;
+    EndLike end;
+    LendingPoolLike aavePool;
+    InterestRateStrategyLike interestStrategy;
+    //RewardsClaimerLike rewardsClaimer;
+    DaiLike dai;
+    DaiJoinLike daiJoin;
+    ATokenLike adai;
+    TokenLike stkAave;
+    SpotLike spot;
+    TokenLike weth;
+    address vow;
+    address pauseProxy;
+
+    bytes32 constant ilk = "DD-DAI-A";
+    D3MHub d3mHub;
+    D3MAavePool d3mAavePool;
+    D3MAavePlan d3mAavePlan;
+    D3MMom d3mMom;
+    D3MOracle pip;
+
+    // Allow for a 1 BPS margin of error on interest rates
+    uint256 constant INTEREST_RATE_TOLERANCE = RAY / 10000;
+    uint256 constant EPSILON_TOLERANCE = 4;
+
+    function setUp() public {
+        emit log_named_uint("block", block.number);
+        emit log_named_uint("timestamp", block.timestamp);
+
+        vat = VatLike(0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B);
+        end = EndLike(0x0e2e8F1D1326A4B9633D96222Ce399c708B19c28);
+        aavePool = LendingPoolLike(0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2);
+        adai = ATokenLike(0x018008bfb33d285247A21d44E50697654f754e63);
+        stkAave = TokenLike(0x4da27a545c0c5B758a6BA100e3a049001de870f5);
```

Comment:
This is unused, so should probably be treated the same as `rewardsClaimer` (probably both commented out with an explainer comment that rewards are turned off).

Reply:
[done](https://github.com/makerdao/dss-direct-deposit/pull/93/commits/a287a962595a66d2e4110d2bfb057c6b0643c1d7)